tools/check_pod_errors

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+<< 'heredoc_pod_error_rule'
+The script checks that no POD error are present in any changed file.
+heredoc_pod_error_rule
+
+libs_files=$(git ls-files lib/ | grep '.pm$' | xargs echo)
+tmpfile=$(mktemp)
+
+success=1
+if test -n "$libs_files"; then
+    for libfile in $libs_files; do
+        perldoc -T -D "${libfile}" 2>/dev/null 1>"$tmpfile"
+        grep -n "POD ERRORS" "$tmpfile" || continue
+
+        success=0
+        error_line=$(grep -n "POD ERRORS" "$tmpfile"|tail -1| cut -f 1 -d ":")
+        echo "ERROR in file ${libfile}"
+        cat "$tmpfile" | sed -n "$error_line,\$"p
+    done
+else
+    echo "No lib files.";
+fi
```

Comment:
Please make sure that the output is compatible with github actions too: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-error-message

Reply:
I guess this is solved, right @foursixnine ?

Reply:
Oh yes 🤩  ![Screenshot 2023-10-25 at 13 41 30](https://github.com/os-autoinst/os-autoinst-distri-opensuse/assets/229240/403f2156-da5b-4399-aa96-b98538684c78)